README.md

```diff
@@ -10,26 +11,25 @@ Use Airflow to author workflows as directed acyclic graphs (DAGs) of tasks.
 The Airflow scheduler executes your tasks on an array of workers while following
 the specified dependencies. Rich command line utilities make performing complex
 surgeries on DAGs a snap. The rich user interface makes it easy to visualize
-pipelines running in production, monitor progress, and troubleshoot issues when
+pipelines running in production, monitor progress, and troubleshoot issues Øhen
 needed.
 
-### Prerequisites
+## Prerequisites
 
 This app is built and deployed with
 [docker](https://docs.docker.com/) and
 [docker-compose](https://docs.docker.com/compose/).
 
-### Dependencies
+### Updating Python dependencies
 
-Most Airflow jobs are thin wrappers that spin up an EMR cluster for running
-the job. Be aware that the configuration of the created EMR clusters depends
-on finding scripts in an S3 location configured by the `SPARK_BUCKET` variable.
-Those scripts are maintained in
-[emr-bootstrap-spark](https://github.com/mozilla/emr-bootstrap-spark/)
-and are deployed independently of this repository.
-Changes in behavior of Airflow jobs not explained by changes in the source of the
-Spark jobs or by changes in this repository
-could be due to changes in the bootstrap scripts.
+Add new Python dependencies into `requirements.in`. Run the following commands with the same Python
+version specified by the Dockerfile.
+
+```bash
+# As of time of writing, python3.7
+pip install pip-tools
+pip-compile
```

Comment:
it's my understanding that `requirements.in` needs to be manually specified here, or it won't actually be used
```suggestion
pip-compile requirements.in
```

Reply:
This is not required, since pip-compile has a set of files that it will look for (`requirements.in` being one of them). No harm in being explicit though.

Reply:
ah, my bad then. no need to include this change.